config/routes.rb

```diff
@@ -14,6 +14,7 @@
   resource :stories do
     post :bulk_destroy, to: "stories#bulk_destroy"
     post :render_markdown
+    patch :update_status
```

Comment:
this would make more sense as a member route instead of a collection route, then we could do `update_status_story_path(@story, state: "approved")` instead of `update_status_path(story_id: @story.id, state: "approved")`

Reply:
I'm also wondering if something like `approve` and `reject` endpoints would be better, so we could say `approve_story_path(@story)`